qa/L0_backend_python/test.sh

```diff
@@ -401,7 +398,6 @@ if [ $? -ne 0 ]; then
   RET=1
 fi
 
-# Disable env test for Jetson since build is non-dockerized and cloud storage repos are not supported
```

Comment:
Maybe we should keep the comment for `cloud storage repos not being supported`? I believe we aren't running the `env` test on jetson.

Reply:
Thanks for catching this, fixed.